packages/react/src/components/Authenticator/shared/__tests__/FormField.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('FormField', () => {
     const describedById = inputField.getAttribute('aria-describedBy');
 
     // now, follow the id this id just like a screen reader would
-    const errors = container.querySelector(`#${describedById}`);
+    const errors = container.querySelector(`[id="${describedById}"]`);
```

Comment:
interesting, I think both of these selectors would be equivalent

Reply:
I think they are but the first one was throwing an invalid selector error, I think because the actual id `amplify-id-:r3:` has special characters